### ISSUE TITLE
[Bug]: Latest version of jaeger doesn't accept open telemetry data on port 4317.

### ISSUE THREAD

**Issue by mpbnkg** [jaegertracing/jaeger#5737]
### What happened?

See this ticket for more info - https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33995

### Steps to reproduce

See this ticket for more info - https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33995

### Expected behavior

See this ticket for more info - https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33995

### Jaeger backend version

latest

**Comment by PedemonteGiacomo** [comment]
Yes I have the same problem that we have already discussed [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33995#issuecomment-2225208526).

With older image version like the 1.58 of the **jaeger-all-in-one** it works fine.

**Comment by yurishkuro** [comment]
Most likely due to recent OTEL Collector change to default receivers to use `localhost` instead of `0.0.0.0`. Please see this fix https://github.com/jaegertracing/jaeger/pull/5734

**Comment by mpbnkg** [comment]
@yurishkuro Yep, it works when I added those two lines :)

**Comment by yurishkuro** [comment]
In the next release we are going back to listening on all IPs.

**Comment by matthewgrossman** [comment]
FYI, this env-based workaround discussed in this issue wasn't working for me; no services external to my jaeger container could send (gRPC) traces to 4317. I noticed in this https://github.com/jaegertracing/jaeger/pull/6134/files that you can do settings overrides, which did work for me:

```
docker run --rm --name jaeger \
  -p 16686:16686 \
  -p 4317:4317 \
  jaegertracing/jaeger:2.0.0 \
  --set=receivers.otlp.protocols.grpc.endpoint="0.0.0.0:4317"
```

Not sure if this is a regression for `2.0`, or if this is just expected behavior now that jaeger has more settings parity with the otel-collector
